### PR TITLE
Add password Spraying tool (SMB, OWA, Lync, Cisco ASA etc...)

### PR DESCRIPTION
**Pull request by Fisjkars**
Add tool : Spray, a password spraying tool for SMB, CISCO , OWA, Lync, VPN

### CODE REVIEW

File: sources/install.sh [hunk 1]
```diff
@@ -2125,6 +2125,13 @@ function install_pcredz() {
   add-test-command "PCredz --help"
 }
 
+function install_spray() {
+  # Password Spraying tool for Active Directory Credentials.
+  colorecho "Installing Spray"
+  git -C /opt/tools/ clone https://github.com/Greenwolf/Spray.git
+  add-aliases spray
+}
```
Comment (ShutdownRepo) [inline sources/install.sh:2133]: Can you add `add-test-command`, as well as `add-history` if you have some example commands at mind?

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]

File: sources/install.sh (hunk 1 above)
Comment (Fisjkars) [reply sources/install.sh:2135]: Hello @ShutdownRepo, normally all the changes you requested has been pushed

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]

File: sources/zsh/history.d/spray [hunk 2]
```diff
@@ -0,0 +1,5 @@
+spray -smb 192.168.0.1 users.txt passwords.txt 1 35 SPIDERLABS
+spray -owa 192.168.0.1 users.txt passwords.txt 1 35 post-request.txt
+spray -lync https://lyncdiscover.spiderlabs.com/ users.txt passwords.txt 1 35
+spray -cisco 192.168.0.1 usernames.txt passwords.txt 1 35
+spray -ovpn 192.168.0.1 943 usernames.txt passwords.txt 1 35
```
[the remark this answers is not in the document]
Comment (Fisjkars) [reply sources/zsh/history.d/spray]: Done ;)

**Review by ShutdownRepo — APPROVED** [review]

**Review by QU35T-code — APPROVED** [review]

**Comment by ShutdownRepo** [comment]
@Fisjkars step 1 of refactor is done, can you change your PR according to the new structure? 
1. sync the latest changed on our `dev` branch to your branch
2. move your changes where they're supposed to end up at

If that's easier for you, you can close this PR and create another one if you'd like

**Pull request closed without merging**